libraries/helpers.rb

```diff
@@ -328,3 +328,27 @@ def check_deprecated_properties
     end
   end
 end
+
+#
+# Chef Stack Helpers
+#
+def prefix
+  (platform_family?('windows') ? 'C:/Chef/' : '/etc/chef/')
+end
```

Comment:
These helpers should be placed in the existing `ChefIngredientCookbook::Helpers` modulespace.